src/NServiceBus.RabbitMQ.Tests/RabbitMqContext.cs

```diff
@@ -61,10 +61,7 @@ public void TearDown()
             channelProvider?.Dispose();
         }
 
-        protected bool TryWaitForMessageReceipt()
-        {
-            return TryReceiveMessage(out var message, incomingMessageTimeout);
-        }
+        protected bool TryWaitForMessageReceipt() => TryReceiveMessage(out var message, incomingMessageTimeout);
```

Comment:
One additional tweak here would be to change `message` to a discard (`_`) since we have no use for the result.